src/Provider/SwiftSmtpTransportProvider.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Provider;
+
+use Psr\Container\ContainerInterface;
+use Yiisoft\Di\Container;
+use Yiisoft\Di\Support\ServiceProvider;
+
+final class SwiftSmtpTransportProvider extends ServiceProvider
+{
+    private string $host;
+    private int $port;
+    private ?string $encryption;
+    private string $username;
+    private string $password;
+
+    public function __construct(
+        string $host = 'smtp.example.com',
```

Comment:
That's not useful default value. Let's remove it.

Reply:
done.